_api-reference/index-apis/create-index.md

```diff
@@ -82,7 +82,14 @@ Setting | Description
 index.number_of_shards | The number of primary shards in the index. Default is 1.
 index.number_of_routing_shards | The number of routing shards used to split an index.
 index.shard.check_on_startup | Whether the index's shards should be checked for corruption. Available options are `false` (do not check for corruption), `checksum` (check for physical corruption), and `true` (check for both physical and logical corruption). Default is `false`.
-index.codec | The compression type to use to compress stored data. Available values are `default` (optimizes for retrieval speed) and `best_compression` (optimizes for better compression at the expense of speed, leading to smaller data sizes on disk). For snapshot distributions built with the sandbox feature enabled, `-Dsandbox.enabled=true`, OpenSearch offers a custom-codecs plugin that supports the value `zstd` for Zstandard compression.
+index.codec |  Determines how the index’s stored fields are compressed and stored on the disk. This setting impacts the size of the index shards and the performance of the operations on the index. Available values are:
+* `default` - This codec employs the 'LZ4' algorithm with a preset dictionary, which prioritizes performance over compression ratio. It offers faster indexing and search operations when compared with 'best_compression', but may result in larger index/shard sizes. If no codec is provided in the index settings, then 'LZ4'  is used as default algorithm for compression.
+* `best_compression` -  This codec utilizes 'zlib' as an underlying algorithm for compression. It achieves high compression ratios resulting in smaller index sizes. However, this may incur additional CPU usage during operations on the index and subsequently may result in high indexing and search latencies. 
```

Comment:
```suggestion
* `best_compression` -  This codec utilizes `zlib` as an underlying algorithm for compression. It achieves high compression ratios resulting in smaller index sizes. However, this may incur additional CPU usage during operations on the index and subsequently may result in high indexing and search latencies. 
```